lib/paperclip/storage/google_drive.rb

```diff
@@ -156,7 +156,7 @@ def public_url_for title
       def search_for_title(title)
         parameters = {
                 'folderId' => find_public_folder,
-                'q' => "title contains '#{title}'", # full_title
```

Comment:
I like the flexibility of the loose matching here, but I do see the benefits of having exact matching as well.  Can you add options to this and `public_url_for` to allow for both?